ext/filter/filter.stub.php

```diff
@@ -2,16 +2,16 @@
 
 /** @generate-function-entries */
 
-function filter_has_var(int $type, string $variable_name): bool {}
+function filter_has_var(int $type, string $var_name): bool {}
 
-function filter_input(int $type, string $variable_name, int $filter = FILTER_DEFAULT, array|int $options = 0): mixed {}
+function filter_input(int $type, string $var_name, int $filter = FILTER_DEFAULT, array|int $options = 0): mixed {}
 
-function filter_var(mixed $variable, int $filter = FILTER_DEFAULT, array|int $options = 0): mixed {}
+function filter_var(mixed $var, int $filter = FILTER_DEFAULT, array|int $options = 0): mixed {}
 
 function filter_input_array(int $type, array|int $options = FILTER_DEFAULT, bool $add_empty = true): array|false|null {}
 
 function filter_var_array(array $data, array|int $options = FILTER_DEFAULT, bool $add_empty = true): array|false|null {}
```

Comment:
```suggestion
function filter_var_array(array $array, array|int $options = FILTER_DEFAULT, bool $add_empty = true): array|false|null {}
```
Per the usual? Though $data is also fine by me.

Reply:
Yes, good idea!

Reply:
And I like the other two suggestions as well!